raster/src/main/scala/geotrellis/raster/package.scala

```diff
@@ -33,20 +33,20 @@ package object raster {
 
   // Keep constant values in sync with macro functions
   @inline final val byteNODATA = Byte.MinValue
+  @inline final val ubyteNODATA = Byte.MinValue & 0xFF
   @inline final val shortNODATA = Short.MinValue
+  @inline final val ushortNODATA = Short.MinValue & 0xFFFF
   @inline final val NODATA = Int.MinValue
   @inline final val floatNODATA = Float.NaN
   @inline final val doubleNODATA = Double.NaN
 
 
-  def isNoData(b: Byte): Boolean = macro NoDataMacros.isNoDataByte_impl
-  def isNoData(s: Short): Boolean = macro NoDataMacros.isNoDataShort_impl
+  // def isNoData(b: Byte): Boolean = macro NoDataMacros.isNoDataByte_impl
+  // def isNoData(s: Short): Boolean = macro NoDataMacros.isNoDataShort_impl
```

Comment:
Not sure if these should be commented out or not. Since isNoData should work with TypeByte and TypeShort.